src/main/scala/molmed/utils/GeneralUtils.scala

```diff
@@ -303,6 +305,12 @@ class GeneralUtils(projectName: Option[String], uppmaxConfig: UppmaxConfig) exte
       else
         ""
 
+    def skiplDuplicated =
+      if(skipDup != -1)
+        "--skip-duplicated --skip-dup-mode " + skipDup
```

Comment:
Is this ok or must the int be cast to a string with e.g. `.toString`? I can't remember exactly what the issue was but I had some problems with the mem allocation without casting... perhaps that was because it was a double which the jvm didn't like.. @Smeds @johandahlberg?